modules/.dependencies.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I'd prefer if there was only one configuration file for this, to make it harder for them to get out of sync, my vote would be for something like

```
"libraries": {
  "video": [<files here>],
  "dependants": [
    "jwPlayerVideoProvider",
     ....
  ]
},
```
IMO it could be put in the same ".submodules.json" file we already have with some refactoring / renaming.